src/tag/index.js

```diff
@@ -25,7 +25,7 @@ class Tag extends PureComponent {
     return (
       <span className={cx('tag')}>
         {label}
-        <button onClick={this.handleClick} className={cx('tag-remove')} type="button">
+        <button onClick={e => this.handleClick(e)} className={cx('tag-remove')} type="button">
```

Comment:
This can be simply `onClick={this.handleClick}` (avoids creating a function on every render). I wonder why linting didn't flag this.

Reply:
Okay. I'll update it.